### PR TITLE
changed json parsing

### PR DESCRIPTION
**Pull request by samtholiya**
## Tests 
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/smartcontract/storagesc/models.go [hunk 1]
```diff
@@ -565,7 +565,7 @@ type StorageAllocation struct {
 	OwnerPublicKey    string                        `json:"owner_public_key"`
 	Stats             *StorageAllocationStats       `json:"stats"`
 	DiverseBlobbers   bool                          `json:"diverse_blobbers"`
-	PreferredBlobbers []string                      `json:"preferred_blobbers"`
+	PreferredBlobbers []string                      `json:"blobbers"`
```
Comment (peterlimg) [inline code/go/0chain.net/smartcontract/storagesc/models.go:568]: Please change the `PreferredBlobbers` too

**Review by peterlimg — CHANGES_REQUESTED** [review]
Is this PR ready or still working in progress?

**Comment by ma2b0043** [comment]
@samtholiya @peterlimg when can we expect to close this PR?

**Comment by samtholiya** [comment]
This pr is not needed so closing this pr

**Pull request closed without merging**